docs/architecture/decisions/0021-django-admin.md

```diff
@@ -34,6 +34,13 @@ In contrast to building an admin interface from scratch where development activi
 involve _building up_, leveraging Django Admin will require carefully _pairing back_ the functionalities available to
 users such as analysts.
 
+On accessibility: Django admin is almost fully accessible out-of-the-box, the expections being tables, checkboxes, and
```

Comment:
```suggestion
On accessibility: Django admin is almost fully accessible out-of-the-box, the exceptions being tables, checkboxes, and
```

Reply:
Finally Rachid has a spelling error and not just me!

Reply:
The expectorations?